libclc/generic/include/clc/async/gentype.inc

```diff
@@ -289,12 +152,6 @@
 #undef __CLC_GENTYPE
 #endif
 
-#define __CLC_GENTYPE float4
-#define __CLC_GENTYPE_MANGLED Dv4_f
-#include __CLC_BODY
-#undef __CLC_GENTYPE_MANGLED
-#undef __CLC_GENTYPE
-
 #define __CLC_GENTYPE float8
```

Comment:
Out of curiosity: why are only some of these types moved?

Reply:
So the ones which had moved were all cases that could use the "optimized" async copy route in the cuda backend (for > sm_80) because they had 4,8, or 16 byte alignment.